scipy/spatial/ckdtree/src/query.cxx

```diff
@@ -133,7 +133,7 @@ struct nodeinfo {
     }
 
     inline void update_side_distance(const int d, const double new_side_distance, const double p) {
-        if (CKDTREE_UNLIKELY(ckdtree_isinf(p))) {
+        if (CKDTREE_UNLIKELY(isinf(p))) {
```

Comment:
In C++ you should use `std::isinf`.

Reply:
Okay, I'll clean this up properly.

This is what it was using before (`ckdtree_isinf` -> `sc_isinf` -> `isinf`) what I did here was just a mechanical replacement.